packages/next/src/client/components/react-dev-overlay/internal/container/RuntimeError/index.tsx

```diff
@@ -182,20 +164,24 @@ export const styles = css`
   }
   [data-nextjs-collapsed-call-stack-details][open]
     [data-nextjs-call-stack-chevron-icon] {
+    transition: transform 0.2s ease;
     transform: rotate(90deg);
   }
+  [data-nextjs-collapsed-call-stack-details] > * {
+    padding-left: var(--size-gap-double);
+  }
   [data-nextjs-collapsed-call-stack-details] summary {
     display: flex;
     align-items: center;
-    margin-bottom: var(--size-gap);
+    padding-left: 0;
```

Comment:
don't indent the summary https://github.com/vercel/next.js/pull/62453/files#r1503540974